spec.html

```diff
@@ -27113,6 +27113,9 @@ <h1>Number.isNaN ( _number_ )</h1>
 
       <emu-clause id="sec-number.issafeinteger">
         <h1>Number.isSafeInteger ( _number_ )</h1>
+        <emu-note>
+          <p>A mathematical integer _n_ is considered "<dfn id="safe-integer">safe</dfn>" if and only if the Number value for _n_ is not the Number value for any other mathematical integer.</p>
```

Comment:
We have a `<dfn>` for integer to mean mathematical values already, so we can drop "mathematical".

```suggestion
          <p>An integer _n_ is considered "<dfn id="safe-integer">safe</dfn>" if and only if the Number value for _n_ is not the Number value for any other integer.</p>
```